File: src/main/java/de/tectoast/emolga/utils/sql/managers/PredictionGameManager.java
package de.tectoast.emolga.utils.sql.managers;

import de.tectoast.emolga.utils.sql.base.DataManager;
import de.tectoast.emolga.utils.sql.base.columns.IntColumn;
import de.tectoast.emolga.utils.sql.base.columns.LongColumn;
import de.tectoast.emolga.utils.sql.base.columns.StringColumn;

public class PredictionGameManager extends DataManager {

    final LongColumn USERID = new LongColumn("userid", this);
    final StringColumn USERNAME = new StringColumn("username", this);
    final IntColumn PREDICTIONS = new IntColumn("predictions", this);



    public PredictionGameManager() {
        super("predictiongame");
        setColumns(USERID, USERNAME, PREDICTIONS);
    }
}

File: src/main/kotlin/de/tectoast/emolga/utils/sql/managers/PredictionGameManager.kt
package de.tectoast.emolga.utils.sql.managers

import de.tectoast.emolga.utils.sql.base.DataManager
import de.tectoast.emolga.utils.sql.base.columns.IntColumn
import de.tectoast.emolga.utils.sql.base.columns.LongColumn
import de.tectoast.emolga.utils.sql.base.columns.StringColumn

object PredictionGameManager : DataManager("predictiongame") {
    private val USERID = LongColumn("userid", this)
    private val USERNAME = StringColumn("username", this)
    private val PREDICTIONS = IntColumn("predictions", this)

    init {
        setColumns(USERID, USERNAME, PREDICTIONS)
    }

    fun addPoint(userid: Long) {
        PREDICTIONS.increment(USERID, userid)
    }
}